README.md

```diff
@@ -23,19 +23,22 @@ Register has been used to test purchasing flows of our [Flagship Reader App](htt
 
 ### Overview
 
-Register is a library and companion app that allows seamless mocking of responses from Google Play's In-app Billing. 
-Register works by implementing the same interface as Google's In-app Billing library [InApp Billing Service](https://github.com/googlesamples/android-play-billing/blob/master/TrivialDrive/app/src/main/aidl/com/android/vending/billing/IInAppBillingService.aidl).
+Register is a companion app and library that allows seamless mocking of responses from Google Play Store Billing. 
+Register works by implementing the same interface as the Google Play Store Billing library [Google Play Billing](https://developer.android.com/google/play/billing/billing_overview).
 
-From a client's perspective, there is no difference in how you work with Google's In-app Billing or Register's implementation.
+You can find an sample app that highlights the features of Google Play Billing here: [Classy Taxi](https://github.com/qinci/android-play-billing/tree/master/ClassyTaxi)
```

Comment:
a

Reply:
I wonder what would happen if we forked classy taxi and added Register

Reply:
We have an upcoming task to update the sample on this repo to more closely mimic the ClassyTaxi sample 👍. At the very least that would be a jumping off point if we thought forking ClassyTaxi would be the right move

Reply:
typo fixed.